JesusFilm/core
# feat: tag refine stack

# Description

### Issue

When you click on a Felt need button or collection button to apply the filter, you are toggling that refinement. But if you apply one and then select another one, you should clear the refinement of the previous, not stack the refinements. This is the current behaviour.

Only problem now is how does a user clear any refinements? I think this needs chips to be completed along with.

[Link to Basecamp Todo](https://3.basecamp.com/3105655/projects)

### Solution

Refine all other refined tags when a new one is clicked

# External Changes

n/a

# Additional information

## How to test
Should be able to select a tag filter and it apply
Once you select another, it should now only search by that filter
